### PR TITLE
DLS-MSD-APP : CREATE MSD HMS UI

### PR DESCRIPTION
**Pull request by darrylkaye**
**Changes:**
<@DLS-MSD-APP-00007> Create HMS UI - Integrate React Bootstrap to MSD app repo

### CODE REVIEW

**Comment by darrylkaye** [comment]
@jcbalang-dev My First Commit.

File: src/pages/projects/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
+
+const Projects = () => {
```
Comment (jcbalang-dev) [inline src/pages/projects/index.js:3]: @darrylkaye nice first insertion

File: package.json
Comment (darrylkaye) [inline package.json:1]: <@DLS-MSD-APP-00007> Create HMS UI - Integrate React Bootstrap to MSD app repo

**Comment by darrylkaye** [comment]
**Changes**
<@DLS-MSD-APP-00010> Create HMS UI - Login Page

**Comment by darrylkaye** [comment]
**Changes**
<@DLS-MSD-APP-00011> Create HMS UI - Admin Dashboard

**Comment by darrylkaye** [comment]
Already merged this changes with PR17.

**Pull request merged**